www/include/reporting/dashboard/viewServicesGroupLog.php

```diff
@@ -120,7 +124,7 @@
 /*
  * Stats Display for selected services group
  */
-if (isset($id) && $id != "NULL") {
+if ($id !== false) {
```

Comment:
this sentence should be merged with the previous one

Reply:
fix with https://github.com/centreon/centreon/pull/8419/commits/23185b9036383d869664a0590531e69abe7f06c2